tests/integration/test_dq_rules.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+import logging
+import shutil
+
+from google.cloud import bigquery
+
+import pytest
+
+from clouddq.integration.bigquery.bigquery_client import BigQueryClient
+from clouddq.main import main
+from clouddq.runners.dbt.dbt_runner import DbtRunner
+from clouddq.runners.dbt.dbt_utils import get_dbt_invocation_id
+from clouddq.utils import working_directory
+
+
+logger = logging.getLogger(__name__)
+
+class TestDqRules:
+
+    @pytest.fixture(scope="session")
+    def client(self):
+        """Get BigQuery Client using discovered ADC"""
+        client = BigQueryClient()
+        yield client
+        client.close_connection()
+
+    @pytest.fixture(scope="session")
+    def create_expected_results_table(
+        self,
+        client,
+        test_resources,
+        gcp_project_id,
+        target_bq_result_dataset_name,
+        target_bq_result_table_name,):
+
+        client = client.get_connection()
+
+        table_id = f"{gcp_project_id}.{target_bq_result_dataset_name}.{target_bq_result_table_name}_expected"
+
+        job_config = bigquery.LoadJobConfig(
+            source_format=bigquery.SourceFormat.CSV,
+            skip_leading_rows=1,
+            autodetect=True,
+            write_disposition="WRITE_TRUNCATE")
+
+        with open(test_resources / "expected_results.csv", "rb") as source_file:
+            job = client.load_table_from_file(source_file, table_id, job_config=job_config)
+
+        job.result()  # Waits for the job to complete.
+
+        table = client.get_table(table_id)  # Make an API request.
+        print(f"Loaded {table.num_rows} rows and {len(table.schema)} columns to {table_id}")
+
+    def test_dq_rules(
+        self,
+        runner,
+        temp_configs_dir,
+        gcp_application_credentials,
+        gcp_project_id,
+        gcp_bq_dataset,
+        gcp_bq_region,
+        target_bq_result_dataset_name,
+        target_bq_result_table_name,
+        tmp_path,
+        client,
+        gcp_impersonation_credentials,
+        gcp_sa_key,
+        create_expected_results_table,
+    ):
+
+        try:
+            temp_dir = Path(tmp_path).joinpath("clouddq_test_cli_integration_4")
```

Comment:
Could you please change path name clouddq_test_cli_integration_4 to something else?